Week-3/Assignment3.3.js

```diff
@@ -0,0 +1,21 @@
+function createIncrement() {
+    let count=0;
+    function increment(){
+        count++;
+        console.log(count);
+    }
+    let message=`Count is ${count}`;
+    function log(){
+        console.log(message);
+    }
+    return[increment,log];
+}
+const[increment,log] =createIncrement(); // count is 0 and message is count is 0
+increment(); //incremented count to 1
+increment(); //incremented count to 2
+increment(); //incremented count to 3
+log();// What is logged?  - Count is 0 because this message string is created only once when createIncrement funtion is called.
```

Comment:
Additional task: i want the count to be printed as 3 inside log method. can you refactor the code?

Reply:
Refactored, Please review.